spec/models/post_mover_spec.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+require_relative "../fabricators/reaction_fabricator.rb"
+require_relative "../fabricators/reaction_user_fabricator.rb"
```

Comment:
```suggestion
```

This two lines are not necessary too since we [load plugins fabricators in core](https://github.com/discourse/discourse/blob/a2000a3559eef9ef17d83ee79cf9a3319eb5df24/spec/rails_helper.rb#L92).